balancer-js/src/modules/subgraph/balancer-v2/Pools.graphql

```diff
@@ -1,23 +1,3 @@
-query Pools(
```

Comment:
This seems to only fetch first 1000 pools? Wasn't sure why this was useful and found it confusing so have removed in favour of using AllPools consistently. Happy to reconsider if needed.

Reply:
I think original idea was to that if you are using a custom query your results can be paginated anyways. Pagination is necessary at some point. I think the only place where we really need all the pools in memory is SOR, which could be a special case.